frontend/pages/api/auth/[...nextauth].ts

```diff
@@ -1,11 +1,12 @@
 import NextAuth, { NextAuthOptions } from 'next-auth'
 import CredentialsProvider from 'next-auth/providers/credentials'
-import { login } from 'api/auth/login'
+import { login } from 'services/auth'
 
 export const authOptions: NextAuthOptions = {
   session: {
     strategy: 'jwt'
   },
+  secret: process.env.NEXTAUTH_SECRET,
```

Comment:
don't forget to add a `.env.sample` or `.env.dev` so you have a reference of expected env vars for this project.

mostly a `nit` for now since you're the only dev here 😆